sourcegraph/sourcegraph
# Cody Web: add required "client-name" param to completions endpoint

https://github.com/sourcegraph/sourcegraph/pull/62048 introduced the minimum supported client version constraint for Cody clients in case Cody Ignore is enabled.

To ensure Cody Web client satisfies this constraint, this PR adds the "client-name" query param to the completions endpoint. The "client-version" param is not required for the web client (web client is of the same version as server) so we omit it. 

## Test plan
- CI
- Tested manually
  - Open chat panel and submit a question
  - Ensure completions endpoint request has the "client-name" query param set to "web"
  - Ensure Cody responds with no error (in particular, look for [this](https://github.com/sourcegraph/sourcegraph/blob/cd900e2de5733c16c20683dba5642e04eabbe047/internal/completions/httpapi/handler.go#L497-L503) error)
